src/SixLabors.Fonts/Tables/General/ColrTable.cs

```diff
@@ -21,7 +21,7 @@ public ColrTable(BaseGlyphRecord[] glyphRecords, LayerRecord[] layers)
 
         public static ColrTable? Load(FontReader reader)
         {
-            using (var binaryReader = reader.TryGetReaderAtTablePosition(TableName))
+            using (BigEndianBinaryReader? binaryReader = reader.TryGetReaderAtTablePosition(TableName))
```

Comment:
Ooh... Hadn't spotted that before. Should be this IMO.

```c#
bool FontReader.TryGetReaderAtTablePosition(TableName, out BigEndianBinaryReader? reader)
```

Reply:
changed with fbcfa78d